low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml

```diff
@@ -0,0 +1,216 @@
+   queue_name: sensor-queue
+   exchange_name: sspl-out
+   routing_key: sensor-key
+   ack_queue_name: sensor-queue
+   ack_exchange_name: sspl-out
+   ack_routing_key: sensor-key
+   username: sspluser
+   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
+   message_signature_username: sspl-ll
+   message_signature_token: ALOIUD986798df69a8koDISLKJ282983
+   message_signature_expires: 3600
+   iem_route_addr:
+   iem_route_exchange_name: sspl-out
+   primary_rabbitmq_host: localhost
+   limit_consul_memory: 50000000
+
+LOGGINGPROCESSOR:
+   virtual_host: SSPL
+   queue_name: iem-queue
+   exchange_name: sspl-in
+   routing_key: iem-key
+   username: sspluser
+   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
+   primary_rabbitmq_host: localhost
+
+RABBITMQCLUSTER:
+   cluster_nodes: localhost
+   erlang_cookie: QLDZYPYEYGHECTHYQXFJ
+
+LOGGINGMSGHANDLER:
+   iem_routing_enabled: false
+   iem_log_locally: true
+
+NODEDATAMSGHANDLER:
+   transmit_interval: 10
+   units: MB
+   disk_usage_threshold: 80
+   cpu_usage_threshold: 80
+   host_memory_usage_threshold: 80
+
+RAIDSENSOR:
+   monitor: true
+   threaded: true
+   RAID_status_file: /proc/mdstat
+
+IPMI:
+   user: admin
+   pass: admin
+
+BMC_INTERFACE:
+   default: system
+
+REALSTORSENSORS:
+   monitor: true
+   polling_frequency: 30
+   polling_frequency_override: 0
+
+REALSTORPSUSENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+REALSTORFANSENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+REALSTORCONTROLLERSENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+REALSTORDISKSENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+REALSTORSIDEPLANEEXPANDERSENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+NODEHWSENSOR:
+   monitor: true
+   threaded: true
+   polling_interval: 30
+
+REALSTORLOGICALVOLUMESENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+REALSTORENCLOSURESENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+IEMSENSOR:
+   threaded: true
+   log_file_path: /var/log/cortx/iem/iem_messages
+   timestamp_file_path: /var/cortx/sspl/data/iem/last_processed_msg_time
+
+SYSTEMDWATCHDOG:
+   monitor: true
+   threaded: true
+   monitored_services:
+   smart_test_interval: 999999999
+   run_smart_on_start: 'False'
+
+NODEHWACTUATOR:
+   ipmi_client: ipmitool
+
+DATASTORE:
+   store_type: file
+   consul_host: 127.0.0.1
+   consul_port: 8500
+
+SASPORTSENSOR:
+  monitor: true
+  threaded: true
+  probe: sysfs
+
+MEMFAULTSENSOR:
+   monitor: true
+   threaded: true
+   probe: sysfs
+
+CPUFAULTSENSOR:
+   monitor: true
+   threaded: true
+   probe: sysfs
+
+STORAGE_ENCLOSURE:
+   mgmt_interface: cliapi
+
+SYSTEM_INFORMATION:
+   setup: cortx
+   cli_type: CS-A
+   data_path: /var/cortx/sspl/data/
+   sspl_log_file_path: /var/log/cortx/sspl/sspl.log
+   log_level: INFO
```

Comment:
sspl_post_install uses ENVIRONMENT & RMQ_CLUSTER keys. Currently its hard code to 'PROD' & 'true' respectively.
Can we add that in this config file?

Reply:
Ok. Can you also give default values of those and do we need to add both in SYSTEM_INFORMATION?
And do we use it apart from sspl_post_install

Reply:
I will take this in PR #279